### PR TITLE
Add deprecation notes

### PR DESCRIPTION
**Pull request by yangc95**
### Description

 Replace paragraph formatting for note formatting when mentioning deprecated features.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by KTom101 — CHANGES_REQUESTED** [review]
@yangc95 Added some minor changes.

File: content/general/concepts/ssl/ssl.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ CatalogContent:
...
+**SSL** stands for **Secure Socket Layer**, and is a cryptographic protocol for securing computer networks.
+
+> **Note**: SSL was deprecated in 1999 and renamed TLS by the [Internet Engineering Task Force](https://www.ietf.org). TLS is essentially an improved version of SSL (it is SSL 3.1) and features a number of improvements that address security concerns that were discovered in the initial and prior implementations of SSL.
 
 SSL uses a cryptographic mechanism featuring a public and private key to encrypt data communicating over internet networks. SSL uses a protocol known as a 'handshake' to determine the authenticity of a server. The main advantage of SSL is the confidence it gives to users that a server has been authenticated by a trusted Certificate Authority (CA). Valid SSL certs can only be issued by accreditted Certificate Authorities which have rigid standards for issuing them.
```
Comment (yangc95) [inline content/general/concepts/ssl/ssl.md:19]:
```suggestion
SSL uses a cryptographic mechanism featuring a public and private key to encrypt data communicating over internet networks. SSL uses a protocol known as a 'handshake' to determine the authenticity of a server. The main advantage of SSL is the confidence it gives to users that a server has been authenticated by a trusted Certificate Authority (CA). Valid SSL certs can only be issued by accredited Certificate Authorities which have rigid standards for issuing them.
```

File: content/general/concepts/ssl/ssl.md [hunk 2]
```diff
@@ -12,7 +12,9 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-**SSL** stands for **Secure Socket Layer**, and is a cryptographic protocol for securing computer networks. SSL was deprecated in 1999 and renamed TLS by the [Internet Engineering Task Force](https://www.ietf.org). TLS is essentially an improved version of SSL (it is SSL 3.1) and features a number of improvements that address security concerns that were discovered in the initial and prior implementations of SSL.
+**SSL** stands for **Secure Socket Layer**, and is a cryptographic protocol for securing computer networks.
```
Comment (yangc95) [inline content/general/concepts/ssl/ssl.md:15]:
```suggestion
**Secure Socket Layer (SSL)** is a cryptographic protocol for securing computer networks.
```

File: content/general/concepts/ssl/ssl.md [hunk 3]
```diff
@@ -12,7 +12,9 @@ CatalogContent:
...
+
+> **Note**: SSL was deprecated in 1999 and renamed TLS by the [Internet Engineering Task Force](https://www.ietf.org). TLS is essentially an improved version of SSL (it is SSL 3.1) and features a number of improvements that address security concerns that were discovered in the initial and prior implementations of SSL.
```
Comment (yangc95) [inline content/general/concepts/ssl/ssl.md:17]:
```suggestion
> **Note**: SSL was deprecated in 1999 and renamed Transport Layer Security (TLS) by the [Internet Engineering Task Force](https://www.ietf.org). TLS is essentially an improved version of SSL (it is SSL 3.1) and features a number of improvements that address security concerns that were discovered in the initial and prior implementations of SSL.
```

File: content/general/concepts/ssl/ssl.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (yangc95) [reply content/general/concepts/ssl/ssl.md]: good idea!

**Review by KTom101 — APPROVED** [review]
@yangc95 LGTM

File: content/html/concepts/elements/terms/font/font.md [hunk 4]
```diff
@@ -14,7 +14,9 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-The **`<font>`** element was used to set the font characteristics of a text. It is now deprecated. Some [CSS](https://www.codecademy.com/resources/docs/css) properties like the [`font`](https://www.codecademy.com/resources/docs/css/typography/font) property should be used instead.
+The **`<font>`** element was used to set the font characteristics of a text.
+
+> **Note** This is now deprecated. Some [CSS](https://www.codecademy.com/resources/docs/css) properties like the [`font`](https://www.codecademy.com/resources/docs/css/typography/font) property should be used instead.
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/font/font.md:19]:
```suggestion
> **Note** This is now deprecated. [CSS](https://www.codecademy.com/resources/docs/css) properties like the [`font`](https://www.codecademy.com/resources/docs/css/typography/font) property should be used instead.
```

**Review by SSwiniarski — APPROVED** [review]
@yangc95 I had one little comment, otherwise it looks good to merge.

**Pull request merged**